morl_baselines/single_policy/esr/eupg.py

```diff
@@ -169,15 +169,21 @@ def update(self):
             next_obs,
             terminateds,
         ) = self.buffer.get_all_data(to_tensor=True, device=self.device)
-        # Scalarized episodic reward, our target :-)
+        # Scalarized episodic reward
         episodic_return = th.sum(rewards, dim=0)
         scalarized_return = self.scalarization(episodic_return)
```

Comment:
This is not used anymore I guess?

Reply:
I used the `scalarized_return` only for logging.